Microsoft.Toolkit.Uwp/Commands/DelegateCommand{T}.cs

```diff
@@ -14,23 +14,34 @@
 using System.Diagnostics;
 using System.Windows.Input;
 
-namespace Microsoft.Toolkit.Uwp.SampleApp.Common
+namespace Microsoft.Toolkit.Uwp.Commands
 {
+    /// <summary>
+    /// Represents a command that can perform a given action for a particular type.
+    /// </summary>
+    /// <typeparam name="T">The parameter type</typeparam>
+    /// <seealso cref="System.Windows.Input.ICommand" />
     public class DelegateCommand<T> : ICommand
```

Comment:
I actually tried it the other way round too, where DelegateCommand inherits from DelegateCommand<T> where T was just object, but that would probably be a breaking change. Although now I think about it, nothing would be using it (as it was in the sample, so wouldn't be a breaking change.